cartographer_ros/src/msg_conversion.cc

```diff
@@ -151,40 +180,30 @@ sensor_msgs::LaserScan ToLaserScan(
 
 sensor_msgs::PointCloud2 ToPointCloud2Message(
     const int64 timestamp, const string& frame_id,
+    const ::cartographer::sensor::PointCloud& point_cloud) {
```

Comment:
use pcl::toRosMsg instead. We use pcl::fromROSMsg in sensor_data_producer.cc already.